doc/lsst.ip.diffim/tasks/lsst.ip.diffim.DipoleMeasurementTask.rst

```diff
@@ -0,0 +1,206 @@
+.. lsst-task-topic:: lsst.ip.diffim.DipoleMeasurementTask
+
+##########################
+DipoleMeasurementTask
+##########################
+The list of badFlags will be used to make a list of keys to check for measurement flags on.  By
+default the centroid keys are added to this list
+
+.. _lsst.ip.diffim.DipoleMeasurementTask-description:
+
+Description
+==================
+
+This class provides a default configuration for running Source measurement on image differences.
+
+.. code-block:: py
+
+    class DipoleMeasurementConfig(SingleFrameMeasurementConfig):
+        "Measurement of detected diaSources as dipoles"
+        def setDefaults(self):
+            SingleFrameMeasurementConfig.setDefaults(self)
+            self.plugins = ["base_CircularApertureFlux",
+                            "base_PixelFlags",
+                            "base_SkyCoord",
+                            "base_PsfFlux",
+                            "ip_diffim_NaiveDipoleCentroid",
+                            "ip_diffim_NaiveDipoleFlux",
+                            "ip_diffim_PsfDipoleFlux",
+                            "ip_diffim_ClassificationDipole",
+                            ]
+            self.slots.calibFlux = None
+            self.slots.modelFlux = None
+            self.slots.instFlux = None
+            self.slots.shape = None
+            self.slots.centroid = "ip_diffim_NaiveDipoleCentroid"
+            self.doReplaceWithNoise = False
+
+These plugins enabled by default allow the user to test the hypothesis that the Source is a dipole.
+This includes a set of measurements derived from intermediate base classes
+DipoleCentroidAlgorithm and DipoleFluxAlgorithm.
+Their respective algorithm control classes are defined in
+DipoleCentroidControl and DipoleFluxControl.
+Each centroid and flux measurement will have _neg (negative)
+and _pos (positive lobe) fields.
+
+The first set of measurements uses a "naive" algorithm
+for centroid and flux measurements, implemented in
+NaiveDipoleCentroidControl and NaiveDipoleFluxControl.
+The algorithm uses a naive 3x3 weighted moment around
+the nominal centroids of each peak in the Source Footprint.  These algorithms fill the table fields
+ip_diffim_NaiveDipoleCentroid* and ip_diffim_NaiveDipoleFlux*
+
+The second set of measurements undertakes a joint-Psf model on the negative
+and positive lobe simultaneously. This fit simultaneously solves for the negative and positive
+lobe centroids and fluxes using non-linear least squares minimization.
+The fields are stored in table elements ip_diffim_PsfDipoleFlux*.
+
+Because this Task is just a config for SingleFrameMeasurementTask, the same result may be achieved by
```

Comment:
Remove these lines.

Reply:
I noticed you already removed the last line in the doc. Do you also mean you want the lobe centroids part of the line removed still?